_episodes/13-scheduler.md

```diff
@@ -355,36 +181,15 @@ Submit the job and monitor its status:
 
 Fantastic, we've successfully changed the name of our job!
 
-> ## Setting up Email Notifications
->
-> Jobs on an HPC system might run for days or even weeks. We probably have
-> better things to do than constantly check on the status of our job with
-> `{{ site.sched.status }}`. Looking at the manual page for
-> `{{ site.sched.submit.name }}`, can you set up our test job to send you an email
-> when it finishes?
->
-> > ## Hint
-> >
-> > You can use the *manual pages* for {{ site.sched.name }} utilities to find
-> > more about their capabilities. On the command line, these are accessed
-> > through the `man` utility: run `man <program-name>`. You can find the same
-> > information online by searching > "man <program-name>".
```

Comment:
Introducing the manual is helpful for future self directed learning, would leave some portion of trying it out as an exercise.

Reply:
Agreed, but
1. `man` is covered by Shell Intro and has therefore fallen out of scope, and
2. these lines are a nested callout in an exercise that was cut in its entirety.